mistralrs-core/src/xlora_models/phi3.rs

```diff
@@ -460,16 +460,12 @@ impl Model {
         tgt_len: usize,
         seqlen_offset: usize,
```

Comment:
This function already passed the `seqlen_offset`. I think it was already being correctly calculated for the current use-case, but I'm not sure